src/o365/spo/commands/list/list-webhook-add.spec.ts

```diff
@@ -475,8 +475,8 @@ describe(commands.LIST_WEBHOOK_ADD, () => {
   });
 
   it('passes validation if the expirationDateTime is in the future but no more than six months from now', () => {
-    const currentDate: Date = new Date();
-    const dateString: string = currentDate.getFullYear() + "-" + (currentDate.getMonth() + 4) + "-01";
+    const fiveDaysFromDate = new Date(new Date().getTime()+(5*24*60*60*1000));
```

Comment:
The test is failing because we exceed the 12 months in a year. We can fixe it more easily, through:

```js
const currentDate: Date = new Date();
currentDate.setMonth(currentDate.getMonth() + 4);
const dateString: string = currentDate.getFullYear() + "-" + currentDate.getMonth() + "-01";
```

This way we retain the original test and let JavaScript add another year when the number of months is >12.

Reply:
Yay this even a better one! No worries happy to jump in anytime :)